src/main/java/com/perficient/movies/exceptions/ErrorResponse.java

```diff
@@ -0,0 +1,14 @@
+package com.perficient.movies.exceptions;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+@Getter
+public class ErrorResponse {
+
+    private String message;
```

Comment:
Are you using these variables?

Reply:
I'm using it in the error handler class.